projects/demo-integrations/cypress/tests/kit/date-time/date-time-separator.cy.ts

```diff
@@ -0,0 +1,49 @@
+import {DemoPath} from '@demo/path';
+
+describe('DateTime | Separator', () => {
+    describe('/', () => {
+        beforeEach(() => {
+            cy.visit(`/${DemoPath.DateTime}/API?separator=/`);
```

Comment:
```suggestion
            cy.visit(`/${DemoPath.DateTime}/API?dateSeparator=/`);
```

Reply:
@vladimirpotekhin there are other `separator => dateSeparator` cases in `*.cy.ts`-files (